frontend/server/src/Controllers/Course.php

```diff
@@ -2782,75 +2752,82 @@ public static function apiRegisterForCourse(\OmegaUp\Request $r): array {
         return ['status' => 'ok'];
     }
 
+    /**
+     * Returns course basic details
+     *
+     * @return array{alias: string, basic_information_required: bool, description: string, name: string, requests_user_information: string}
+     */
+    private static function getBasicCourseDetails(
+        \OmegaUp\DAO\VO\Courses $course,
+        \OmegaUp\DAO\VO\Identities $identity
+    ) {
+        return [
+            'name' => strval($course->name),
+            'description' => strval($course->description),
+            'alias' => strval($course->alias),
+            'basic_information_required' => boolval(
+                $course->needs_basic_information
+            ),
```

Comment:
```suggestion
            'basic_information_required' => $course->needs_basic_information,
```

según yo esa propiedad ya es un booleano.

Reply:
Cierto, se quita el `boolval`